app/code/community/Bolt/Boltpay/Model/Cron.php

```diff
@@ -42,6 +42,8 @@ class Bolt_Boltpay_Model_Cron
      *
      * As an artifact, we leave the parent quotes, (i.e. all Magento created quotes), and converted
      * immutable quotes. We delegate cleanup responsibility of these to the merchants.
+     *
+     * @codeCoverageIgnore Not really testable. For one thing sfo.entity_id is the primary key, so should never be NULL.
```

Comment:
@adentong This is a decent temporary solution.   I will write a unit test for this next week.

Reply:
Ah yes you're definitely right.

Reply:
Another difficulty I found is whenever I try to update the updated time to more than two weeks ago it gets automatically updated to the current time. I couldn't figure out how to disable the automatic updating of updated time.

Reply:
I would imagine that if no triggers were defined in the DB, a direct query would be the approach to solve this.  You probably tried that, but, I will keep this in mind if I get to solving this on Monday before you do.

Reply:
I have in fact not tried a direct query. Thanks for informing me of that possibility.